ios/Sources/HomeFeature/QuestionnaireView.swift

```diff
@@ -0,0 +1,43 @@
+import Styleguide
+import SwiftUI
+
+public struct QuestionnaireView: View {
+    public let onTapAnswer: () -> Void
```

Comment:
could we make any format for tapAction name? 

ref: https://github.com/DroidKaigi/conference-app-2021/issues/446
> Implement action with callbacks. Such as **tapAction: @escaping () -> Void.